Add hippod_login.py to hippod/hippod

muthuubalakan opened the pull request:
The hippod_login system handle the hippod login, checks user credentials
and set cookie.
hippod_login.Login() can be imported to hippod run.py

hgn commented on hippod/hippod_login.py:
```diff
@@ -0,0 +1,173 @@
+
+import os
+
```
No newlines here for standard includes

hgn commented on hippod/hippod_login.py:
```diff
@@ -0,0 +1,173 @@
+
+
+# We use ERROR for server_error web response.
+ERROR = '''<!DOCTYPE html>
```
No, this is a static error page, should no assets where all HTML files are stored

hgn commented on hippod/hippod_login.py:
```diff
@@ -0,0 +1,173 @@
+
+    """
+
+    def _check_the_file(self, filename):
```
Public methods first, the the private ones. This is the flow, think new readers of the source code: you want to start to see the big pictures, then you want to go into the details. Right?

muthuubalakan committed:
Add hippod_login.py to hippod/hippod

The hippod_login system handle the hippod login, checks user credentials
and set cookie.
hippod_login.Login() can be imported to hippod run.py

muthuubalakan committed:
Add login_system: Contains files required for login.

login_system has following files.
1. configuration.json
2. login.html
3. redirect.html

muthuubalakan committed:
Change assets name to config. Name conflict with app/assets

This name cause an error with app/assets where all static files, js and css
are stored.

The general idea here is all js, and css files must have stored in
a separate folder and route static files together as aiohttp add_static
requires.
